### DISCUSSION TITLE
Movie Template: remove the "min" from the Duration using {{VALUE:Runtime}}?

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [chhoumann/quickadd#316] [Q&A] [answered]
The plugin itself works flawlessly, but when I use it to add a movie the {{VALUE:Runtime}} returns a duration with "min" after it, which is understandable. However, somehow my appears to read the "min" as millions, I think, I messes with a chart I make using that data.

I'm not sure how to address that exactly, but would it somehow be possible to get the data without the "min"? Not sure if this is the case, but thought I'd ask.

Thanks!

**Comment by chhoumann** [comment, marked as answer]
Sure! You can add `Runtime: selectedShow.Runtime.replace(" min", ""),` to the `QuickAdd.variables` assignment in the script. Like this:

```diff
	QuickAdd.variables = {
		...selectedShow,
+      Runtime: selectedShow.Runtime.replace(" min", ""),
		actorLinks: linkifyList(selectedShow.Actors.split(",")),
		genreLinks: linkifyList(selectedShow.Genre.split(",")),
		directorLink:
			selectedShow.Director === "N/A"
				? " "
				: linkifyList(selectedShow.Director.split(",")),
		fileName: replaceIllegalFileNameCharactersInString(selectedShow.Title),
		typeLink: `[[${selectedShow.Type === "movie" ? "Movies" : "Series"}]]`,
	};
```

**Reply by ReaderGuy42** [reply]
Thanks, but this doesn't work, any other ideas?

**Reply by ReaderGuy42** [reply]
I either did it wrong last time or something changed because now it works :)

**Reply by chhoumann** [reply]
Great to hear!
